packages/mobile/src/escrow/saga.ts

```diff
@@ -108,13 +108,17 @@ function* withdrawFromEscrow(action: EndVerificationAction) {
     const escrow: Escrow = yield call(getEscrowContract, web3)
     const account: string = yield call(getConnectedUnlockedAccount)
     const inviteCode: string = yield select((state: RootState) => state.invite.redeemedInviteCode)
+    const tmpWalletPrivateKey = inviteCode
```

Comment:
why not just rename the above var instead of copy it?

Reply:
True :D
Done